src/libraries/Initializable.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.13;
+
+
+contract Initializable {
+    bool public initialized = false;
```

Comment:
Should this be a private member with a corresponding getter?  (Sorry, I know you said you're regretting that decision. :))